src/html.rs

```diff
@@ -28,6 +28,8 @@ struct SrcSetItem<'a> {
     descriptor: &'a str,
 }
 
+const WHITESPACES: &'static [char] = &['\t', '\n', '\x0c', '\r', ' '];
```

Comment:
Do we need `\x0b` here by any chance as well?  Python's `string.whitespace` includes `\x0b` and https://doc.rust-lang.org/reference/whitespace.html is even wider, but the reference spec https://infra.spec.whatwg.org/#ascii-whitespace doesn't include it, so we're probably fine.

I also think it needs to be renamed to something like `SRCSET_WHITESPACES` since it's only used for srcset, and the file is html.rs (all things related to HTML).

Reply:
Hi @snshn,

>so we're probably fine.

seems yes, `\x0b` is out of scope HTML5 spec. 

> I also think it needs to be renamed to something like SRCSET_WHITESPACES since it's only used for srcset, and the file is html.rs (all things related to HTML).

But on the other hand, while it's currently used ONLY for `srcset`, the `WHITESPACES` list makes sense for whole HTML5 spec. I would name it `HTML5_WHITESPACES`, but as you mentioned it's already defined inside `html.rs`,  I ignored the prefix. So if you still insist on renaming, let me know I'll do it.

Reply:
If it's fart of the HTML5 spec, I guess it can be just `WHITESPACES` in this file.

Reply:
Let's move it below `ICON_VALUES` though, to keep it alphabetical?